### DISCUSSION TITLE
VSCODE Master css plugin code color become white

### DISCUSSION THREAD

**Discussion by jackyyangtw** [master-co/css#65] [Question]
I  found that when using { } in class, the following code will become white, is any setting to fix this? Or I misuse something and I didn't find out?  Thanks!

![螢幕擷取畫面 2022-09-20 130645](https://user-images.githubusercontent.com/50864464/191172157-439d7fa6-175e-4dcf-b09b-7ad3424b2623.png)

**Comment by 1aron** [comment]
@jackyyangtw Can you provide the HTML code?

**Reply by jackyyangtw** [reply]
```
<section class="sec-1">
    <div class="container">
        <div class="row">
            <div class="col-12">
                <h2 class="t:center">Protection from Informational Incidents</h2>
            </div>
            <div class="col-md-5 flex jc:center ai:center">
                <ul class="{flex;pb:1.5em}_li {mr:0.5em;f:18}_div">
                    <li>
                        <div>‧</div>
                        <div>Certified to <span class="font:bold inline-block">ISO 27001</span> since 2014</div>
                    </li>
                    <li>
                        <div>‧</div>
                        <div>A member of <span class="font:bold">TWCERT</span> and <span class="font:bold">CVE</span></div>
                    </li>
                    <li><div>‧</div>Significantly reduced risk of harm or loss due to data misuse, leakage, tampering and destruction due to human error.</li>
                    <li><div>‧</div>Proactive security incident reduction during operation.</li>
                </ul>
            </div>
            <div class="col-md-7">
                <object data="https://webdls.ieiworld.com/data/landing-page/Information-Security-Policy/Information_Security_Policy_SVG-v1.svg" type=""></object>
            </div>
        </div>
    </div>
</section>
```

**Reply by 1aron** [reply]
@jackyyangtw We found that the problem caused by `{flex;pb:1.5em}_li {mr:0.5em;f:18}_div` will be fixed asap! Thx.
You can fix it first by wrapping the line:
```html
<ul class="
    {flex;pb:1.5em}_li
    {mr:0.5em;f:18}_div
">
```

**Reply by jackyyangtw** [reply]
Thanks, it works!

**Reply by 1aron** [reply]
@jackyyangtw Fixed. Just update and reload your Master CSS Language Service!

<img width="664" alt="Screen Shot 2022-09-20 at 4 59 05 PM" src="https://user-images.githubusercontent.com/33840671/191215302-5be5956b-655c-4c56-8cb8-2a9cf424c9e4.png">

**Reply by jackyyangtw** [reply]
I updated, and I found this will cause the following code become white, too. By the way @width1200、@width2560 are my custom breakpoints.
```
<figure class="fade-in-item left:80 top:48% left:163@width1200 {left:10;top:46%}@width2560">
    <img class="w:200 w:300@width2560" src="../_img/P8-7.png" alt="">
</figure>
```
![螢幕擷取畫面 2022-09-21 141619](https://user-images.githubusercontent.com/50864464/191428750-cd3b311f-d655-4d2a-b2d1-5a6a97fe3160.png)

**Reply by 0Miles** [reply]
@jackyyangtw We are fixing it.

This issue can be caused by the last character in the group, "%".
To avoid this problem, you can temporarily swap left:10 and top:46%.
```
{top:46%;left:10}@width2560
```

**Reply by 1aron** [reply]
@jackyyangtw Now fixed in v0.2.5. Just reload again. 😌
